fdbserver/Ratekeeper.actor.cpp

```diff
@@ -1278,9 +1278,22 @@ ACTOR Future<Void> ratekeeper(RatekeeperInterface rkInterf, Reference<AsyncVar<S
 				if (p.totalTransactions > 0) {
 					self.smoothReleasedTransactions.addDelta( req.totalReleasedTransactions - p.totalTransactions );
 
+					// TODO process commitCostEstimation
 					for(auto tag : req.throttledTagCounts) {
 						self.throttledTags.addRequests(tag.first, tag.second);
 					}
+					for (auto& tag : req.throttledTagCommitCostEst) {
```

Comment:
Nit: This loop can be simplified by using `for (const auto &[tagName, cost] : req.throttledTagCommitCostEst) { ... }`. Then you won't have to use `first` and `second`.

Reply:
Good tip. I've learned.